### PR TITLE
[keymap] Add narze keymap for usb-usb converter

### PR DESCRIPTION
**Pull request by narze**
## Description

Add `usb_usb/keymap/narze` layout

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/usb_usb/keymaps/narze/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# narze's layout for usb-usb converter
+
+## Key features
+- Qwerty + [Colemak](https://colemak.com) layouts, and you can type Qwerty on software-level Colemak as well. Very useful for gaming or when your friend wanna type something but don't use Colemak.
+- [(S)uper (D)uper Mode](https://github.com/jasonrudolph/keyboard#super-duper-mode) inspired by [jasonrudolph](https://github.com/jasonrudolph), with [some extensions](https://gist.github.com/narze/861e2167784842d38771) such as backspace & forward delete.
+
+## Build instructions
+- `cd /path/to/qmk_firmware`
+- Ensure latest libraries are loaded `make git-submodule`
+- Build & flash : `make converter/usb_usb:narze:dfu
```
Comment (noroadsleft) [inline keyboards/converter/usb_usb/keymaps/narze/README.md]: Closing the inline code block.

```suggestion
- Build & flash : `make converter/usb_usb:narze:dfu`
```

File: keyboards/converter/usb_usb/keymaps/narze/rules.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```
Comment (fauxpark) [inline keyboards/converter/usb_usb/keymaps/narze/rules.mk]: You can reduce this down to:

```c
EXTRAKEY_ENABLE = no
NKRO_ENABLE = yes
COMBO_ENABLE = yes
```

File: keyboards/converter/usb_usb/keymaps/narze/README.md (hunk 1 above)
Comment (drashna) [reply keyboards/converter/usb_usb/keymaps/narze/README.md]: Also:
```suggestion
- Build & flash : `make converter/usb_usb:narze:flash`
```

File: keyboards/converter/usb_usb/keymaps/narze/rules.mk (hunk 2 above)
Comment (narze) [reply keyboards/converter/usb_usb/keymaps/narze/rules.mk]: 👌

**Pull request merged**